test/clean-local-registry-tag.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+if [[ $1 == *"docker.io"* || $1 == *"quay.io"* ]]; then
```

Comment:
Would we ever hit this? (just a question, it's fine to have it). `print-node-ip` should always return an IP, right?

Reply:
@aanm asked for this, I think it's sensible to have it, but we probably won't ever hit this.